Shradha/JavaScript/Assignment3/ques1-2-3.html

```diff
@@ -0,0 +1,142 @@
+                }
+            }
+        </script>
+        <hr>
+        <div>
+            <h4 class="m-3">Ques 3: Even or Odd Number</h4>
+        </div>
+        <div class="row m-4">
+            <label class="col-sm-2"><b>Enter the Number:</b></label>
+            <div class="col-sm-10">
+                <input type="number" id="num1" placeholder="Enter number" required>
+                <button id="evenoddnum" onclick="evenodd()" class="btn btn-success p-1 ml-lg-2" style="width: 150px;" type="button">Click Me !!!</button>
+            </div>
+            <p class="ml-1 p-2" id="pid2"></b></p>
+        </div>
+        <div class="row m-4">
+            <label class="col-sm-2"><b>Enter the Number:</b></label>
+            <div class="col-sm-10">
+                <input class="mt-2" type="number" id="num2" placeholder="Enter number" required>                
+                <button id="evenoddnum" onclick="evenodd1()" class="btn btn-success p-1 ml-lg-2" style="width: 150px;" type="button">Click Me !!!</button>
+            </div>
+            <p class="ml-1 p-2" id="para2"></b></p>
+        </div>
+        <script>
+            function evenodd()
+            {
+                var x, num = true ;
+                x = document.getElementById("num1").value ;
+                for (i=0; i<=x; i++)
+                {
+                    if (x % 2 == 1)
+                    {
+                        num = false ;
+                        break; 
+                    }
+                }
+                if(num)
+                {
+                    document.getElementById("pid2").innerHTML ="<b>Result: </b>" + x + " is an even number";
+                }
+                else
+                {
+                    document.getElementById("pid2").innerHTML = "<b>Result: </b>" + x + " is an odd number";
+                }
+            }
+        </script>
+        <script>
+            function evenodd1(){
+                var x, num = true ;
+                x = document.getElementById("num2").value ;
+                for (i=0; i<=x; i++)
+                {
+                    if (x % 2 == 1) 
+                    {
+                        num = false ;
+                        break; 
+                    }
+                }
+                if(num)
+                {
+                    document.getElementById("para2").innerHTML = "<b>Result: </b>" + x + " is an even number";
+                }
+                else
+                {
+                    document.getElementById("para2").innerHTML = "<b>Result: </b>" + x + " is an odd number";
+                }
+            }
+        </script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    </body>
```

Comment:
no need to write multiple script tags. Place all the functions in one script tag in the head section

Reply:
ok mam